presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMaterializedViewLogicalPlanner.java

```diff
@@ -1077,6 +1089,551 @@ public void testBaseToViewConversionWithGroupBy()
+                                                    ImmutableMap.of("sum_71", functionCall("sum", ImmutableList.of("count"))),
+                                                    PARTIAL,
+                                                    aggregation(
+                                                            ImmutableMap.of("count", functionCall("count", ImmutableList.of("count_72"))),
+                                                            FINAL,
+                                                            node(
+                                                                    ExchangeNode.class,
+                                                                    anyTree(
+                                                                            aggregation(
+                                                                                    singleGroupingSet("ds"),
+                                                                                    ImmutableMap.of(Optional.of("count_72"), functionCall("count", ImmutableList.of())),
+                                                                                    ImmutableList.of(),
+                                                                                    ImmutableMap.of(),
+                                                                                    Optional.empty(),
+                                                                                    PARTIAL,
+                                                                                    node(
+                                                                                            ProjectNode.class, constrainedTableScan(
+                                                                                                    table,
+                                                                                                    ImmutableMap.of("ds", singleValue(createVarcharType(10), utf8Slice("2021-07-12"))),
+                                                                                                    ImmutableMap.of("ds", "ds"))))))))),
+                                    anyTree(
+                                            constrainedTableScan(
+                                                    view,
+                                                    ImmutableMap.of("ds", singleValue(createVarcharType(10), utf8Slice("2021-07-11"))),
+                                                    ImmutableMap.of())))));
+
+            assertPlan(getSession(), viewQuery, expectedPattern);
+            assertPlan(queryOptimizationWithMaterializedView, baseQuery, expectedPattern);
+        }
+        finally {
+            queryRunner.execute("DROP MATERIALIZED VIEW IF EXISTS " + view);
+            queryRunner.execute("DROP TABLE IF EXISTS " + table);
+        }
+    }
+
+    @Test
+    public void testBaseToViewConversionCountOutputAsExpected()
```

Comment:
Does this test the same things as testBaseToViewConversionCountOptimizationWithStitching

Reply:
The plan tree is similar, but this tests the output . I could remove the `assertPlan` call, but I don't think the output testing is useless personally.